### PR TITLE
{bio}[GCCcore/7.3.0,dummy/,foss/2018b] CheckM v1.0.13, prodigal v2.6.3, pplacer v1.1.alpha19

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3135.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/d76ea65a252fb792d99f2f48f737551f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2572.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3f671471286753da81fea914ffdd8a90 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2038.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e825b4550cfd3196d102aceb3e0d9dbf for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
rescomp2 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8b18bebd091cf811509959e03472830d for a full test report.

File: easybuild/easyconfigs/p/pplacer/pplacer-1.1.alpha19.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+version = '1.1.alpha19'
+
+homepage = 'http://matsen.fhcrc.org/pplacer/'
+description = """Pplacer places query sequences on a fixed reference phylogenetic tree to maximize phylogenetic
+ likelihood or posterior probability according to a reference alignment. Pplacer is designed to be fast, to give
+ useful information about uncertainty, and to offer advanced visualization and downstream analysis."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# compiler pplacer from source requires an ancient OCaml version,
```
Comment (verdurin) [inline easybuild/easyconfigs/p/pplacer/pplacer-1.1.alpha19.eb]: @boegel Don't want to be 'that guy'. However, it should be 'compiling'...

Comment (boegel) [reply easybuild/easyconfigs/p/pplacer/pplacer-1.1.alpha19.eb]: fixed in #7885

**Pull request merged**